packages/sij-dialect-postgresql/src/ast/util.ts

```diff
@@ -13,6 +17,83 @@ type PgUtil =
   | Unlisten
   | Vacuum;
 
+/*
+FETCH [ direction ] [ FROM | IN ] cursor_name
+*/
+interface Fetch
+  extends Tagged<
+    'Fetch',
+    {
+      readonly direction: DirectionOption;
+      readonly cursorName: Ident;
+    }
+  > {}
+const Fetch = (args: UnTag<Fetch>): Fetch => tag('Fetch', args);
+
+/*
+MOVE [ direction ] [ FROM | IN ] cursor_name
+*/
+interface Move
+  extends Tagged<
+    'Move',
+    {
+      readonly direction: DirectionOption;
+      readonly cursorName: Ident;
+    }
+  > {}
+const Move = (args: UnTag<Move>): Move => tag('Move', args);
+
+type DirectionOption =
+  | 'Next'
+  | 'Prior'
+  | 'First'
+  | 'Last'
+  | Absolute
+  | Relative
+  | 'All'
+  | 'Forward'
```

Comment:
I would make these duplicate cases into one case with the `count` as `NumLit | null`.

Reply:
To be more general, would you prefer if this list had the minimal number of options necessary to cover all cases? For example `Next`,  and `Forward`, are equivalent to `Forward 1` or `Relative 1`. I think I could get rid of `Next`, `Prior`, `First`, `Last`,  `All`, `Forward`, and `Backward`. That would leave `Absolute count`, `Relative count`, `Forward count`, `Backward count`, `ForwardAll`, and `BackwardAll`.

Reply:
It's not clear to me which you're seeing as the duplicate cases.

Reply:
Oh I see it, we could actually do `count: NumLit | 'All' | null`